package/upgrade/upgrade_manifests.sh

```diff
@@ -1257,6 +1257,73 @@ sync_containerd_registry_to_rancher() {
   kubectl annotate --overwrite=true setting.harvesterhci.io containerd-registry "harvesterhci.io/upgrade-patched=$REPO_HARVESTER_VERSION"
 }
 
+upgrade_rbac() {
+
+  # only versions before v1.3.0 that upgrading to v1.3.0 need this patch
+  if [[ ! "${UPGRADE_PREVIOUS_VERSION%%-rc*}" < "v1.3.0" ]]; then
+    echo "Only versions before v1.3.0 need this patch."
+    return
+  fi
+
+  full_name=$(kubectl get rolebinding -o wide |grep "harvesterhci.io:cloudprovider" |awk '{print $4}')
```

Comment:
Do we need to add the `-A` flag here? Assuming the user created the guest cluster on a namespace different from the "default" NS.

The other one is, is it possible to have multiple results after grepping? That could cause the following logic to break. Thanks!

Reply:
I think you are correct. I need to handle more than one guest cluster case.

Reply:
updated! Add `-A` flags and also handle multiple items